Add Slite integration documentation (#583)
Promptless/docs: src/content/docs/docs/configuring-promptless/context-sources/slite.mdx

---
title: Slite
slug: docs/configuring-promptless/context-sources/slite
sidebar:
  hidden: false
  order: 19
---
import Note from '@components/fern/Note.astro';

Slite integration provides **read-only** access to your team's knowledge base for documentation automation. When Slite is configured as a context source, Promptless can search your Slite notes for relevant information when creating suggestions—but it never modifies your Slite content.

> ## How It Works as Context Source

Reviewer: ```suggestion
## How It Works as Context
```

When Promptless picks up the merged PR, it searches your Slite workspace for related notes and resolves any Slite URLs referenced in the PR description. It reads the PRD and carries those agreed-upon details into the documentation suggestion—the same feature name, the same retry limits, and the same rationale your team already settled on—instead of inferring them from the diff alone. The result is a docs PR that matches the spec your product and engineering teams wrote, not a re-derived approximation.

## Configuration

Slite only supports API key authentication rather than OAuth. To connect Slite:

1. In your Slite account, go to **Settings > API** and create a new API key. The API key grants Promptless read access to all notes visible to the account that created the key. We recommend using a service account with appropriate access scope.
2. Go to the [Integrations page](https://app.gopromptless.ai/integrations) in your Promptless dashboard.
3. Click **Connect** on the Slite integration card.

> 4. Paste your API key in the modal and click Connect.

Reviewer: ```suggestion
4. Enter your API key and click **Connect**.
```

## Data Privacy

> Promptless queries Slite data in real-time and does not store any of your Slite content. All searches are performed on-demand when documentation updates are needed. Query results are cached briefly to improve performance during a single documentation session, then discarded.

Reviewer: ```suggestion
Promptless queries Slite data in real-time and doesn't store any of your Slite content. All searches are performed on-demand when documentation updates are needed. Query results are cached briefly to improve performance during a single documentation session, then discarded.
```

> Promptless only reads from Slite—it never creates, updates, or deletes notes.

Reviewer: ```suggestion
Promptless only reads from Slite. Promptless never creates, updates, or deletes notes.
```
